packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -241,6 +253,57 @@ export const createRDSCluster = async (config: RDSConfig): Promise<ClusterInfo>
   }
 };
 
+/**
+ * Accesses the local RDS Aurora serverless V2 cluster with one DB instance using the given input configuration.
+ * @param config Configuration of the database cluster.
+ * @returns EndPoint address, port and database name of the accessed RDS cluster.
+ */
+export const useRDSCluster = async (config: RDSConfig): Promise<ClusterInfo> => {
+  try {
+    // Send requests to get the cluster and instance config info
+    const databaseName = 'defaultdb';
+    const identifier = config.identifier;
+    const instanceIdentifier = createInstanceIdentifier(config.identifier);
```

Comment:
We cannot assume that the instances developer have in their cluster adhere to this naming. You can get the instance attached to cluster from your describe cluster command.